mungegithub/features/aliases.go

```diff
@@ -119,8 +108,9 @@ func (a *Aliases) EachLoop() error {
 }
 
 // RegisterOptions registers options used by the Aliases feature.
-func (a *Aliases) RegisterOptions(opts *options.Options) {
+func (a *Aliases) RegisterOptions(opts *options.Options) sets.String {
```

Comment:
maintainer nit: `RegisterOptions`' comment is inconsistent.. some files (like this one) don't describe the return at all, some say `returns any options that should trigger a restart when they are changed`, others say `returning any that are immutable`